rush.json

```diff
@@ -440,6 +440,11 @@
       "packageName": "@uifabric/test-utilities",
       "projectFolder": "packages/test-utilities",
       "versionPolicyName": "lockedMajor"
+    },
+    {
+      "packageName": "@uifabric/azure-themes",
+      "projectFolder": "packages/azure-themes",
+      "shouldPublish": false
```

Comment:
Actually you should probably comment out the linkage to `azure-themes` in `customizations.ts` until this value is `true` and this package is being published correctly. Otherwise it could break anybody who syncs.

Reply:
(It shouldn't cause too many issues as the only package that uses it is internal, but that's how I staged it out just to be safe.)

Reply:
This is actually an awesome catch/insight. I agree with Jason that it is best to not have that link until you're ready for it to publish.

We can in fact publish with the first checkin (so you don't have to do this busywork of staging it into 2 PRs) but today you'll need to sync with @JasonGore to manually do the initial publish before you hit the merge button as we have not fixed the pipeline bug w.r.t. initial publish.